setup.bat

```diff
@@ -0,0 +1 @@
+mvn package
```

Comment:
I second this, I don't think it hurts anyone to have Windows build scripts along with the Linux ones.